### PR TITLE
#376 Implement sync Poster and Database for Tables and tests

### PR DESCRIPTION
**Pull request by ebercamargo**
#376 Implement sync Poster and Database for Tables and tests

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#458](https://github.com/timelesslounge/timelessis/issues/458) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#458](https://github.com/timelesslounge/timelessis/issues/458) is assigned to @vryazanov[/z](https://www.0crat.com/u/vryazanov), here is [why](https://www.0crat.com/footprint/CF9LG1YCA/6a719bb3-a102-4cd8-b295-fa327943e5f8); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: timeless/reservations/views.py [hunk 1]
```diff
@@ -92,45 +100,55 @@ def get(self, company_id):
         return jsonify(reservations_json)
 
 
-@bp.route("/")
-def list_reservations(reservations):
+@bp.route("/create", methods=("GET", "POST"))
```
Comment (vryazanov) [inline timeless/reservations/views.py]: @ebercamargo We have a generic view  https://github.com/timelesslounge/timelessis/blob/master/timeless/views.py#L242 
let's use it

File: tests/test_reservation.py [hunk 2]
```diff
@@ -1,8 +1,9 @@
 from datetime import datetime
 from http import HTTPStatus
 
+import  pytest
 
+@pytest.mark.skip(reason="authentication not implemented for reservations")
```
Comment (vryazanov) [inline tests/test_reservation.py]: @ebercamargo it's actually implemented. see https://github.com/timelesslounge/timelessis/blob/master/tests/integration/it_companies_test.py#L21

File: timeless/reservations/views.py [hunk 3]
```diff
@@ -92,45 +100,55 @@ def get(self, company_id):
...
+        "reservations/create_edit.html", action="edit",
         id=id
     )
 
 
-@bp.route("/delete", methods=["POST"])
-def delete():
-    flash("Delete not yet implemented")
+@bp.route("/delete/<int:id>", methods=["POST"])
+def delete(id):
```
Comment (vryazanov) [inline timeless/reservations/views.py]: @ebercamargo We have generic `DeleteView`, let's use it instead of custom solution

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @vryazanov[/z](https://www.0crat.com/u/vryazanov), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#458](https://github.com/timelesslounge/timelessis/issues/458) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa)

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +20 point(s) just awarded to @vryazanov[/z](https://www.0crat.com/u/vryazanov)

**Comment by 0crat** [comment]
Quality review completed: +6 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request closed without merging**